screenbuf/screenbuf.go

```diff
@@ -26,7 +26,7 @@ type ScreenBuf struct {
 	height int
 }
 
-// New creates and initializes a new ScreenBuf.
+// New creates and initializes a new ScreenBuf struct.
```

Comment:
adding extra type information like `struct` or `error` isn't done usually.

Reply:
Nice to know, I'll remove that.